scripts-available/CDB_CartodbfyTable.sql

```diff
@@ -906,12 +906,12 @@ BEGIN
   -- so we check to see if that's the case, and rename it out of the way
   IF has_usable_primary_key AND has_usable_pk_sequence THEN
     -- See if the existing sequence is squatting on our preferred name
-    destseq := Format('%s_%s_seq', relname, const.pkey);
+    destseq := Format('tmp_%s_%s_seq', relname, const.pkey);
```

Comment:
But now you're checking if the `tmp_*` seq exists (see comparison with relseq below). This seems wrong to me.